### ISSUE TITLE
Debug bar replaces parameter placeholder in database name and columns names

### ISSUE THREAD

**Issue by FacedSID** [yiisoft/yii2#8592]
Debug bar replaces parameter placeholder in whole SQL statement, even in database name and columns names.

The code below:
```php
$command = $db->createCommand('SELECT COUNT(*) FROM `addresses_aliases` WHERE `address_id` = :address_id AND `alias` = :alias', [
    'address_id' => $address_id,
    'alias'      => $alias
]);

$command->execute();
```

will be processed by debug bar as:

```SQL
SELECT `addresses`.* FROM `addresses` left join `addresses_'Санкт-Петербург, Курортный район, Приморское шоссе'es` ON `addresses_'Санкт-Петербург, Курортный район, Приморское шоссе'es`.address_id = `addresses`.id WHERE addresses_'Санкт-Петербург, Курортный район, Приморское шоссе'es.`'Санкт-Петербург, Курортный район, Приморское шоссе'` = :'Санкт-Петербург, Курортный район, Приморское шоссе'
```

as you can see, debug bar replaced table name `addresses_aliases` to `addresses_'Санкт-Петербург, Курортный район, Приморское шоссе'es` and even column name to the parameter value.

If i'll change my code to follows:
```php
$command = $db->createCommand('SELECT COUNT(*) FROM `addresses_aliases` WHERE `address_id` = :address_id AND `alias` = :alias', [
    ':address_id' => $address_id,
    ':alias'      => $alias
]);

$command->execute();
```
debug bar will replace placeholders by the right way:
```SQL
SELECT `addresses`.* FROM `addresses` left join `addresses_aliases` ON `addresses_aliases`.address_id = `addresses`.id WHERE addresses_aliases.`alias` = 'Санкт-Петербург, Курортный район, Приморское шоссе'
```

**Comment by Alex-Code** [comment]
If you use param binding you need to prefix your array keys with the colon `:` character.
```php
[
    ':param' => $value
]
```

**Comment by FacedSID** [comment]
Ok. But, would you fix it?

**Comment by Alex-Code** [comment]
I can't see anything as needing to be fixed.
If your params don't have the colon prefix I wouldn't think your queries would work.

**Comment by klimov-paul** [comment]
Param prefixing with ':' is not necessary.
This issue is not about SQL execution - it about wrong SQL presentation given by debug panel.
It should be fixed using `preg_replace()` instead of `strtr()`.

**Comment by klimov-paul** [comment]
Issue resolved by commit 0d48f10
